Scripts/Python/ki/xKIChat.py

```diff
@@ -707,13 +707,14 @@ def DisplayStatusMessage(self, message, netPropagate=0):
     ## Adds a player to the recent players folder.
     def AddPlayerToRecents(self, playerID):
 
-        vault = ptVault()
-        PIKAFolder = vault.getPeopleIKnowAboutFolder()
+        isAdded = False
+        PIKAFolder = ptVault().getPeopleIKnowAboutFolder()
         if PIKAFolder:
             PIKA = PIKAFolder.upcastToPlayerInfoListNode()
```

Comment:
I see now where the redundant upcast came from -_-

Reply:
Yeah, sorry. I was taking from what I had seen used before and I didn't know to be skeptical... :(